### ISSUE TITLE
Can't build any more with Tarpaulin coverage

### ISSUE THREAD

**Issue by nuxeh** [time-rs/time#267]
I'm getting the following error:

```
$ cargo tarpaulin -- --verbose
[INFO tarpaulin] Creating config
[INFO tarpaulin] Running Tarpaulin
[INFO tarpaulin] Building project
   Compiling semver v0.9.0
   Compiling serde_json v1.0.56
   Compiling toml v0.5.6
   Compiling string_cache v0.8.0
   Compiling serde_urlencoded v0.6.1
   Compiling toml v0.4.10
   Compiling docopt v1.1.0
   Compiling time v0.2.10
   Compiling hyper v0.13.6
   Compiling selectors v0.22.0
   Compiling serde_rusqlite v0.14.0
   Compiling rustc_version v0.2.3
error: could not compile `time`.

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
[ERROR tarpaulin] Failed to compile tests! Error: cannot find attribute `skip` in this scope
Error: "Failed to compile tests! Error: cannot find attribute `skip` in this scope"
```

**Comment by jhpratt** [comment]
Duplicate of #265. Most development is happening on the v0.3 branch nowadays.

**Comment by nuxeh** [comment]
I guess this may have been fixed in 0.2.16, however, since I have run tarpaulin successfully where the latest `time` version is a dependency, so I'll close.

**Comment by nuxeh** [comment]
I see you've already done this, the page hadn't updated.
